packages/augur-ui/config/webpack.common.config.js

```diff
@@ -103,12 +107,71 @@ module.exports = {
         ]
       },
       {
-        test: /\.css$/,
+        test: /\.s?css$/,
```

Comment:
why are all these rules needed

Reply:
They're to allow webpack to process orbit-web's code.

Reply:
why isn't orbits already built and available as a package?

Reply:
We would have to build it separately. That won't happen from our usual build process so we'd either need to:
1) Add a step to our usual process. Before someone does UI work, they'd need to run `yarn workspace orbit-web build`.
2) Add that command to `augur-ui/package.json` under the `build` commands.

Reply:
I’m confused. Doesnt the top level build build everything then build augur-ui?

Could orbits-web just get built by the top level build process. 
How does augur-sdk make it in augur-ui’s node_modules?

Could orbits-web follow the same process?
Would we basically have to make a node module out of orbits-web like augur-sdk in order for augur-ui to use it?

Reply:
> I’m confused. Doesnt the top level build build everything then build augur-ui?

The top-level `build` only builds typescript, it doesn't run webpack or call the `build` script of any of its packages. We have to run `build` for augur-ui ourselves, though usually we use the `dev` wrapper instead.

> Could orbits-web just get built by the top level build process.

It can but that means changing the package.json script for `build` from just building typescript to also calling orbit-web's `build` command. This is what I had augur-ui's `build` script doing in an earlier PR.

> How does augur-sdk make it in augur-ui’s node_modules?
> Could orbits-web follow the same process?
> Would we basically have to make a node module out of orbits-web like augur-sdk in order for augur-ui to use it?

This would involve importing orbit-web into our project. That would violate our isolation policy of keeping 3rd party software in iframes.

We'd also need to turn it into a typescript-compatible project. When I tried to get around using babel by using our typescript loader, I got errors even with `transpileOnly` set to `true`. I believe this mean that they use features provided by babel but not typescript. Their .babelrc has enough plugins that this wouldn't surprise me.

Reply:
Oh! Another possibility is to build orbit-web completely separately but commit its output somewhere, probably orbit-web/dist. Our webpack file would still need to copy things over but that's probably the only change it would need.